packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -855,6 +856,13 @@ export class AzureCommunicationCallAdapter<AgentType extends CallAgent | BetaTea
     });
   }
 
+  /* @conditional-compile-remove(reaction) */
+  public async onReactionClicked(emoji: string): Promise<void> {
```

Comment:
we should call the param here: `reaction` especially since if Contoso is to provide their own sprites like talked about in option one we should make sure that we are making them map to that as well.